src/wp-includes/rest-api/endpoints/class-wp-rest-template-revisions-controller.php

```diff
@@ -160,18 +160,23 @@ public function register_routes() {
 	 *
 	 * @since 6.4.0
 	 *
-	 * @param int $parent_post_id Supplied ID.
+	 * @param string $parent_post_id Supplied ID.
```

Comment:
I just fixed this clear mistake in https://core.trac.wordpress.org/changeset/58156, while the rest is still being discussed.